utils/measure/measure.py

```diff
@@ -299,7 +395,38 @@ def should_resume(self, csv_file_path: str) -> bool:
 
 
     def get_resume_variation(self, csv_file_path: str) -> Variation:
```

Comment:
Actually because you added these docs I noticed this return type is incorrect.

```suggestion
    def get_resume_variation(self, csv_file_path: str) -> Variation | None:
```

Reply:
So I guess there's value about adding type's to the docs ;P